### DISCUSSION TITLE
Call for volunteers | Mapping TOMP API to beckn

### DISCUSSION THREAD

**Discussion by ravi-prakash-v** [beckn/mobility#54] [Announcements]
TOMP is an open standard used in Europe to allow booking of tickets for public transit systems. We need someone to help us with mapping this API to beckn protocol.

**Comment by bharath-ganapathy** [comment]
Link to the swaggerhub documentation of TOMP API: https://app.swaggerhub.com/apis/TOMP-API-WG/transport-operator_maas_provider_api/1.3.0

**Comment by singhs020** [comment]
@core-wg-admin I would love to help.

**Comment by ravi-prakash-v** [comment]
Hi @singhs020,thank you for taking this initiative. You are a true beckn open collective **_champion_**! We shall get in touch with you shortly regarding the same. Can you please DM me your email address on Discord so that we can share any relevant information regarding the same?

**Comment by bharath-ganapathy** [comment]
Adding a video recording of an overview of the TOMP API:
https://drive.google.com/file/d/1MbCGheIKvKPr4ql3saZ9iWgp-45O7e7R/view?usp=sharing
